Samples/iOS-Swift/PerformanceBenchmarks/SentrySDKPerformanceBenchmarkTests.m

```diff
@@ -0,0 +1,134 @@
+#import "SentryProcessInfo.h"
+#import <XCTest/XCTest.h>
+#import <objc/runtime.h>
+
+// To get around the 15 minute timeout per test case on Sauce Labs.
+static NSUInteger SentrySDKPerformanceBenchmarkTestCases = 5;
+static NSUInteger SentrySDKPerformanceBenchmarkIterationsPerTestCase = 4;
+
+// All results are aggregated to analyse after completing the separate,
+// dynamically generated test cases
+static NSMutableArray *allResults;
+static BOOL checkedAssertions = NO;
+
+@interface SentrySDKPerformanceBenchmarkTests : XCTestCase
+
+@end
+
+@implementation SentrySDKPerformanceBenchmarkTests
+
+/**
+ * Dynamically add a test method to an XCTestCase class.
```

Comment:
`l`: A few sentences describing why we do this would be excellent, so you don't have to read the whole article. Furthermore, I think we should point out that because we dynamically add tests we do this in ObjC.

Reply:
This is already explained [earlier in the file](https://github.com/getsentry/sentry-cocoa/pull/1926/files#diff-79d11845c2567088706849360b62a59fc0827501503856f11b867fa220e25508R5) and in [`//develop-docs/README.md`](https://github.com/getsentry/sentry-cocoa/pull/1926/files#diff-5c546014845a9fa5c3f1d7c177d42608d616f3206059c5efac54404ffcc390e8R47), i've added a part to the earlier comment to direct to the readme, and added more info in the readme.